src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj

```diff
@@ -14,6 +14,7 @@
         This is a public preview release of the Azure HDInsight SDK. Fixed the namespace of the HDInsight management client.
       ]]>
 		</PackageReleaseNotes>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
What is the intention here?
@shahabhijeet do we allow this?

Reply:
The intention is to fix build errors because of duplicate attributes in our AssemblyInfo, e.g. "Duplicate 'System.Reflection.AssemblyConfigurationAttribute' attribute Management.HDInsight(netstandard1.4)". Let me know if there is a better way.

Reply:
This change is not required. Please try the following and let us know if the issue persists.
- Undo the change
- Pull latest changes from upstream
- Run `msbuild build.proj`
- Run `msbuild build.proj /t:build /p:Scope=SDKs/HDInsight`

Reply:
Thanks, reverted.